src/tests/ftest/datamover/posix_preserve_props.py

```diff
@@ -61,7 +61,7 @@ def run_dm_preserve_props(self, tool, cont_type, api):
         self.preserve_props_path = join(self.tmp, "cont_props.h5")
 
         # Create a source cont
-        cont1 = self.create_cont(pool1, cont_type=cont_type)
+        cont1 = self.get_container(pool1, type=cont_type)
```

Comment:
(pylint-unexpected-keyword-arg) Unexpected keyword argument 'type' in method call